app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -1432,6 +1490,8 @@ AdvancedFilter.propTypes = {
   authenticity_token: PropTypes.string,
   advancedFilterUpdate: PropTypes.func,
   updateStickySettings: PropTypes.bool,
+  jurisdiction_paths: PropTypes.object,
```

Comment:
these props also need to be provided to the instance of the AdvancedFilter component from the PatientsFilters component (used for custom exports)

Reply:
same thing for `assigned_users`

Reply:
My bad, done!